Modern/wrapperPlugins/src/main/java/org/bonej/wrapperPlugins/ThicknessWrapper.java

```diff
@@ -138,8 +141,15 @@ public void run() {
 			resultsTable = SharedTable.getTable();
 		}
 		if (showMaps) {
+			LUT fire = Common.makeFire();
 			trabecularMap = thicknessMaps.get(true);
+			StackStatistics trabecularStats = new StackStatistics(trabecularMap);
+			trabecularMap.setDisplayRange(0.0, trabecularStats.max);
+			trabecularMap.setLut(fire);
 			spacingMap = thicknessMaps.get(false);
+			StackStatistics spacingStats = new StackStatistics(spacingMap);
+			spacingMap.setDisplayRange(0.0, spacingStats.max);
+			spacingMap.setLut(fire);
```

Comment:
Variables should be `final`

Reply:
> we should write a test with mocked UI to see that LUT assignment works OK. I can help with that.

Yes please, could you do it and show me and @alessandrofelder how it works

Reply:
Maybe adding a helper method is borderline here, but in principle creating methods is not only about avoiding repetition, it's also about making code easier to understand. At best it can make code self-documenting. Consider:
```java
public void myMethod() {
    // Do thing 1
    ...
   // Calculate thing 2
   double x;
   ...
   x = Math.sqrt(y + z);
   // Try thing 3
   try {
       ...
   } 
   catch(final IOException e) {
       ...
   }   
}
```
Versus:
```java
public void myMethod() {
    doThing1();
    final double x = calculateThing2();
    doThing3();
}
```